docs/UserGuide.adoc

```diff
@@ -12,32 +12,33 @@ ifdef::env-github[]
 :tip-caption: :bulb:
 :note-caption: :information_source:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level3
+:repoURL: https://github.com/AY1920S1-CS2103T-T13-1/main
 
-By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+By: `CS2013T-T13-1`      Since: `Sep 2019`      Licence: `MIT`
 
 == Introduction
 
-AddressBook Level 3 (AB3) is for those who *prefer to use a desktop app for managing contacts*. More importantly, AB3 is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps. Interested? Jump to the <<Quick Start>> to get started. Enjoy!
+ORGANice Transplant Manager(OTM) is a desktop application for hospital administration staff +
+to easily manage kidney transplants.
+With OTM, you can organise information about patients, doctors, and kidney donors, +
```

Comment:
remove kidney?

Reply:
I have rectified this in my latest commit.

Reply:
OTM or just ORGANice?

Reply:
I'm conflicted with this also, should we have an acronym for this or just refer to the app as 'ORGANice' all the way